src/index.html

```diff
@@ -17,11 +17,696 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="/src/styles/index.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo__image"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link active"
```

Comment:
There is an inconsistency in the class naming for active navigation links. The first link uses 'is-active' while the fourth link uses 'active'. It's important to maintain consistency in class naming conventions to avoid confusion and potential styling issues.